requirements/common.txt

```diff
@@ -1,11 +1,19 @@
+beautifulsoup4==4.6.0
+bs4==0.0.1
+certifi==2017.11.5
+chardet==3.0.4
+defusedxml==0.5.0
+Django==1.10.8
```

Comment:
Django is included in the more specific files (e.g. `django110.txt`) and should not be included here.